Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+@Library('shared-libraries') _
+pipeline{
+  agent {label 'devExpLinuxPool'}
+  environment{
+    JAVA_HOME_DIR="/home/builder/java/openjdk-1.8.0-262"
```

Comment:
Is there a later version of Java that we can use in Jenkins? I'm on openjdk 1.8.0_382 locally.

Reply:
I didn't find the openJDK package. This is the latest version that we downloaded. I'll check if Progress team has any other sources.